src/components/Timer/timer.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import Timer from 'react-compound-timer';
+
+interface TimerProps {
+  isRecording: boolean;
+}
+
+interface TimerMethods {
+  start: () => void;
+  stop: () => void;
+  reset: () => void;
+}
+
+export class RecordingTimer extends React.Component<TimerProps> {
+
+  timerController({ start, stop, reset }: TimerMethods) {
+    if (this.props.isRecording) {
+      start();
+    }
+    if (!this.props.isRecording) {
+      stop();
+    }
+  }
```

Comment:
Not sure about the flow of logic here. Can you think of another way to write it?